src/coreclr/vm/method.cpp

```diff
@@ -2797,8 +2797,9 @@ PCODE MethodDesc::GetPortableEntryPoint()
 {
     WRAPPER_NO_CONTRACT;
 
-    // The portable entry point is currently the same as the
-    // temporary entry point.
+    if (HasStableEntryPoint())
+        return GetStableEntryPoint();
```

Comment:
Is this still needed? What is the problem that it is fixing?

Reply:
Okay, I think I've figured this out.